functions/README.md

```diff
@@ -13,10 +13,35 @@ Getting Started
 ---------------
 
 - [Add Firebase to your iOS Project](https://firebase.google.com/docs/ios/setup).
+- [Set up Firebase Functions](https://firebase.google.com/docs/functions/get-started)
+Functions in the docs should be updated, since the functions applied in the
+Quickstart might not be the same as the ones in the example from the doc above.
 
-### Google Sign In Setup
-- In Xcode, [add a custom URL scheme for your reversed client ID](https://developers.google.com/identity/sign-in/ios/start-integrating).
-- You can find this in the `GoogleService-Info.plist`
+To get aligned with the Quickstart, the `index.js` could be updated to 
```

Comment:
To align with the Quickstart, the `index.js` should be updated to

Reply:
also, `quickstart` should be lowercase (here and throughout).